<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShellPage</class>
 <widget class="QWidget" name="ShellPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>538</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>&lt;b&gt;Configure Shell&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line14">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="shellLinenoCheckBox">
       <property name="toolTip">
        <string>Select whether line numbers margin should be shown.</string>
       </property>
       <property name="text">
        <string>Show Line Numbers Margin</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="shellCTEnabledCheckBox">
       <property name="toolTip">
        <string>Select this to enable calltips</string>
       </property>
       <property name="text">
        <string>Calltips Enabled</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="shellWordWrapCheckBox">
       <property name="toolTip">
        <string>Select to enable wrapping at word boundaries</string>
       </property>
       <property name="text">
        <string>Word Wrap Enabled</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="shellACEnabledCheckBox">
       <property name="toolTip">
        <string>Select this to enable autocompletion</string>
       </property>
       <property name="text">
        <string>Autocompletion Enabled</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="shellSyntaxHighlightingCheckBox">
       <property name="toolTip">
        <string>Select to enable syntax highlighting</string>
       </property>
       <property name="text">
        <string>Syntax Highlighting Enabled</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>History</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="textLabel1_20">
        <property name="text">
         <string>max. History Entries:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="shellHistorySpinBox">
        <property name="toolTip">
         <string>Enter the number of history entries allowed</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>343</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Navigation Style:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="shellHistoryStyleComboBox">
        <property name="toolTip">
         <string>Select the history style</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QCheckBox" name="shellHistoryWrapCheckBox">
        <property name="toolTip">
         <string>Select to wrap around while navigating through the history</string>
        </property>
        <property name="text">
         <string>Wrap around while navigating</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QCheckBox" name="shellHistoryCursorKeysCheckBox">
        <property name="toolTip">
         <string>Select to make Up- and Down-keys move in history</string>
        </property>
        <property name="whatsThis">
         <string>&lt;b&gt;Up/Down keys navigate in history&lt;b&gt;
&lt;p&gt;Select this entry to make Up- and Down-keys navigate in history. If unselected history navigation may be performed by Ctrl-Up or Ctrl-Down.&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string>Up/Down keys navigate in history</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Font</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="monospacedFontButton">
        <property name="toolTip">
         <string>Press to select the font to be used as the monospaced font</string>
        </property>
        <property name="text">
         <string>Monospaced Font</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="monospacedFontSample">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Monospaced Text</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="monospacedCheckBox">
        <property name="toolTip">
         <string>Select, whether the monospaced font should be used as default</string>
        </property>
        <property name="text">
         <string>Use monospaced as default</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="linenumbersFontButton">
        <property name="toolTip">
         <string>Press to select the font for the line numbers</string>
        </property>
        <property name="text">
         <string>Line Numbers Font</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="marginsFontSample">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>2345</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="stdOutErrCheckBox">
     <property name="toolTip">
      <string>Select to show debuggee stdout and stderr</string>
     </property>
     <property name="text">
      <string>Show stdout and stderr of debuggee</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>shellLinenoCheckBox</tabstop>
  <tabstop>shellACEnabledCheckBox</tabstop>
  <tabstop>shellSyntaxHighlightingCheckBox</tabstop>
  <tabstop>shellWordWrapCheckBox</tabstop>
  <tabstop>shellCTEnabledCheckBox</tabstop>
  <tabstop>shellHistorySpinBox</tabstop>
  <tabstop>shellHistoryStyleComboBox</tabstop>
  <tabstop>shellHistoryWrapCheckBox</tabstop>
  <tabstop>shellHistoryCursorKeysCheckBox</tabstop>
  <tabstop>monospacedFontButton</tabstop>
  <tabstop>monospacedCheckBox</tabstop>
  <tabstop>linenumbersFontButton</tabstop>
  <tabstop>stdOutErrCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
